system/Security/Exceptions/SecurityException.php

```diff
@@ -11,9 +11,10 @@
 
 namespace CodeIgniter\Security\Exceptions;
 
+use CodeIgniter\Exceptions\ExceptionInterface;
 use CodeIgniter\Exceptions\FrameworkException;
 
-class SecurityException extends FrameworkException
+class SecurityException extends FrameworkException implements ExceptionInterface
```

Comment:
I'm sure FrameworkException already implements ExceptionInterface, so there is absolutely no need to reimplement it here. Please revert changes made to this file.